doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -73,56 +82,40 @@ The definition above is very application centric and look at cloud native from t
+- **Kubernetes-based Application / VNF Management**: similar to VNFM, this is an application layer capability that is out of scope of CNTT. This includes Kubernetes-based Application Package Management, such as Helm, as this is a client application and set of libraries that would be part of a modern/cloud native VNFM, not part of the infrastructure itself.
 
+<p align="center"><img src="../figures/ch01_scope_k8s.png" alt="Kubernetes Reference Architecture scope" title="Kubernetes Reference Architecture scope" width="100%"/></p>
+<p align="center"><b>Figure 1-1:</b> Kubernetes Reference Architecture scope</p>
 
+<a name="1.5"></a>
+## 1.5 Approach
 
+The approach taken in this Reference Architecture is to start simple (i.e. with a basic Kubernetes architecture), and then add detail and additional features/extensions as is required to meet the requirements of the Reference Model.
 
-## General thoughts
-***Please note that the notes below will be removed and replaced with content in the chapters of this Reference Architecture***
-
-Kubernetes itself is a “system for automating deployment, scaling, and management of containerized applications” and therefore Kubernetes place within our architecture should be closely linked to the application lifecycle (this is especially important when considering bare-metal containerisation). However, it is very important to point out that a CaaS also consist of other open source projects, or add-ons, such as:
-- CNI-compliant network plugins
-- CSI-compliant storage plugins
-- CRI-compliant container runtimes
-- service mesh options
-- service discovery options
-- monitoring and logging options
-- etc.
-
-Figure 1-1 below shows how Kubernetes, CaaS, and the CaaS Manager might fit within the current ETSI architecture.  In terms of this Reference Architecture, we should focus on the requirements of the new interfaces and the capabilities within the CaaS Manager and the Kubernetes clusters.
-
-<p align="center"><img src="../figures/ch01_k8s_arch.png" alt="Kubernetes Architecture for NFV" title="Kubernetes Architecture for NFV" width="100%"/></p>
-<p align="center"><b>Figure 1-1:</b> Kubernetes Architecture for NFV</p>
-
-### A note on virtualised and containerised workloads
-It is possible that a single VNFM will be managing applications (or components) that are virtualised (i.e. running in VMs) and applications (or components) that are containerised (i.e. running in containers), at the same time.  There are different approaches to achieving this:
-1. Application manager (EMS, VNFM) uses both IaaS (VIM) API and Kubernetes API - the former for VM based workloads, the latter for containerised workloads.
-2. Application manager (EMS, VNFM) uses just the Kubernetes API for both VM-based and container-based workloads, with the Kubernetes PaaS managing the lifecycle of VMs using on of the following methods:
-    a) Kubernetes interacts with IaaS/VIM API (Cluster API model)
```

Comment:
Are there any solutions for this?

Reply:
@CsatariGergely for the removed content do you mean?  Given this is removed for now, do we need to change anything?

Reply:
Oh, yes sorry for the removed content. No need to change.